src/topology/vector_bundle.lean

```diff
@@ -249,3 +249,138 @@ lemma is_topological_vector_bundle_is_topological_fiber_bundle :
+topological_fiber_bundle_core.mem_triv_change_source ↑Z i j p
+
+variable (ι)
+
+/-- Topological structure on the total space of a topological bundle created from core, designed so
+that all the local trivialization are continuous. -/
+instance to_topological_space : topological_space (total_space Z.fiber) :=
+topological_fiber_bundle_core.to_topological_space ι ↑Z
+
+variables {ι} (b : B) (a : F)
+
+@[simp, mfld_simps] lemma coe_cord_change (i j : ι) :
+  topological_fiber_bundle_core.coord_change ↑Z i j b = Z.coord_change i j b := rfl
+
+/-- Extended version of the local trivialization of a fiber bundle constructed from core,
+registering additionally in its type that it is a local bundle trivialization. -/
+def local_triv (i : ι) : topological_vector_bundle.trivialization R F Z.fiber :=
+{ linear := λ x hx, { map_add := λ v w, by simp only [topological_fiber_bundle_core.local_triv_ext,
+      local_homeomorph.to_fun_eq_coe, coe_fst, coe_snd_map_apply,
+      topological_fiber_bundle_core.local_triv_apply, linear_map.map_add, coe_cord_change],
+  map_smul := λ r v, by simp only [topological_fiber_bundle_core.local_triv_ext, coe_fst,
+    local_homeomorph.to_fun_eq_coe, topological_fiber_bundle_core.local_triv_apply, coe_cord_change,
+    coe_snd_map_smul, linear_map.map_smul] },
+  ..topological_fiber_bundle_core.local_triv_ext ↑Z i }
+
+@[simp, mfld_simps] lemma mem_local_triv_source (i : ι) (p : total_space Z.fiber) :
+  p ∈ (Z.local_triv i).source ↔ p.1 ∈ Z.base_set i :=
+iff.rfl
+
+/-- Preferred local trivialization of a vector bundle constructed from core, at a given point, as
+a bundle trivialization -/
+def local_triv_at (b : B) : topological_vector_bundle.trivialization R F Z.fiber :=
+Z.local_triv (Z.index_at b)
+
+lemma mem_source_at : (⟨b, a⟩ : total_space Z.fiber) ∈ (Z.local_triv_at b).source :=
+by { rw [local_triv_at, mem_local_triv_source], exact Z.mem_base_set_at b }
+
+@[simp, mfld_simps] lemma local_triv_at_apply : ((Z.local_triv_at b) ⟨b, a⟩) = ⟨b, a⟩ :=
+topological_fiber_bundle_core.local_triv_at_apply Z b a
+
+instance : topological_vector_bundle R F Z.fiber :=
+{ inducing := λ b, ⟨ begin refine le_antisymm _ (λ s h, _),
+  { rw ←continuous_iff_le_induced,
+    exact topological_fiber_bundle_core.continuous_total_space_mk ↑Z b, },
+  { refine is_open_induced_iff.mpr ⟨(Z.local_triv_at b).source ∩ (Z.local_triv_at b) ⁻¹'
+      (Z.local_triv_at b).base_set.prod s, (continuous_on_open_iff
+      (Z.local_triv_at b).open_source).mp (Z.local_triv_at b).continuous_to_fun _
+      (is_open.prod (Z.local_triv_at b).open_base_set h), _⟩,
+    rw [preimage_inter, ←preimage_comp, function.comp],
+    simp only [id.def],
+    refine ext_iff.mpr (λ a, ⟨λ ha, _, λ ha, ⟨Z.mem_base_set_at b, _⟩⟩),
+    { simp only [mem_prod, mem_preimage, mem_inter_eq, local_triv_at_apply] at ha,
+      exact ha.2.2, },
+    { simp only [mem_prod, mem_preimage, mem_inter_eq, local_triv_at_apply],
+      exact ⟨Z.mem_base_set_at b, ha⟩, }} end⟩,
```

Comment:
you're missing an indentation here (the proof should be strictly to the right of `inducing` and `locally_trivial`).